scrapers_next/la/people.py

```diff
@@ -151,11 +151,73 @@ def process_item(self, item):
         return LegislatorDetail(p, source=URL(detail_link, timeout=30))
 
 
-class Senate(Legislators):
+class HouseLegislators(HtmlListPage):
+    selector = CSS("fieldset div.card-body")
+
+    def process_item(self, item):
+        name_dirty = CSS("a").match(item)[0].text_content().strip()
+        print(name_dirty)
```

Comment:
Take out un-necessary `print` statements. (Either use `self.warning`/`self.info`/`self.debug` or just remove it entirely).

Reply:
i somehow always a couple print statements!! thanks